projects/1.20/assets/weapons-of-miracles-epicfight/wom/lang/zh_cn.json

```
@@ -0,0 +1,183 @@
+	"item.wom.netherite_belt" : "下界合金腰带",
+	"item.wom.netherite_chains" : "下界合金锁链",
+
+	"item.wom.agony" : "苦难之刺",
+	"item.wom.agony.tooltip" : "一把轻盈且锋利的长枪，能使出致命的一击。跳跃攻击能够跳越2个方块。",
+	"skill.wom.agony_plunge" : "天跃俯冲",
+	"skill.wom.agony_plunge.tooltip" : "“在俯冲攻击敌人之前向空中闪现。”\n攻击敌人将为俯冲攻击充能最多3次\n最终伤害将根据能量次数进行倍增。",
+	
+	"item.wom.tormented_mind" : "折磨巨斧",
+	"item.wom.tormented_mind.tooltip" : "用于大肆杀戮的武器\n这把斧头的持有者将遭受其前任持有者的愤怒所带来的痛苦，并将其传递给下一位持有者。",
+	"skill.wom.true_berserk" : "真实狂怒",
+	"skill.wom.true_berserk.tooltip" : "“愿神明敬畏你······”\n折磨巨斧的愤怒将使战技充能值无法阅读。",
+
+	"item.wom.ruine" : "毁灭之剑",
+	"item.wom.ruine.tooltip" : "一把古老的武器，锻造于被遗忘的知识之中，并注入了末影人的力量。\n其平衡和设计在战斗中能够实现最高的精度与效率。",
+	"skill.wom.plunder_perdition" : "夺魂",
+	"skill.wom.plunder_perdition.tooltip" : "“执行末影仪式，窃取敌手精元。”\n\n该技能可从1层至多叠加到10层。\n冲刺或跳跃时的技能总是可用，但技能效果会根据堆叠的数量发生变化。2层时，你将在短时间内获得较低的移速。3层时，你将根据你的位置和方向传送至最后击中的目标。\n10层时，可以在行走或站立时使用技能，将剑插入地面并在你周围的所有敌人下方召唤破土而出的利剑。\n每击中一个敌人将使你恢复5%的最大耐力和1点生命值（每个横扫之刃等级增加1点生命值回复）。\n然后，你将获得10秒的整体速度提升（每个横扫之刃等级增加10秒持续时间），速度提升的强弱取决于击中敌人的数量。",
+
+	"item.wom.satsujin" : "杀戮之刃",
+	"item.wom.satsujin.tooltip" : "这把刀中封印着恶魔。\n它强化了这把刀并增加其效力，但由于恶魔仍然被封印，它无法提供更多的力量……\n\n被动：在4秒不攻击后自动纳刀。在拔刀攻击结束后的较短时间内也会自动纳刀。",
+	"skill.wom.sakura_state" : "斩樱形态",
+	"skill.wom.sakura_state.tooltip" : "在绝对的宁静中斩出至锐的一击。\n向前方释放一次樱花斩。在奔跑时释放变为樱花冲刺斩，快速穿越身前的敌人并斩击他们。技能释放次数可以叠加总共8次。\n每拥有一个横扫之刃魔咒等级，将增加1/3的初始叠加次数。\n在首次释放樱花斩/樱花冲刺斩后，你将进入斩樱形态。\n如果在4秒内不再进行樱花斩/樱花冲刺斩，将失去所有的技能释放次数并退出斩樱形态。樱花斩/樱花冲刺斩后将自动纳刀。\n冲刺攻击和樱花冲刺斩将在敌人身上叠加3次绽樱创痕（每拥有一个横扫之刃等级额外+1次）。\n如果在1.5秒内不继续施加冲刺攻击和樱花冲刺斩，叠加的所有绽樱创痕将对目标造成持续地斩击。",
+
+	"item.wom.ender_blaster" : "末影爆裂铳",
+	"item.wom.ender_blaster.tooltip" : "这把武器已经超越了时空，寻找可以控制其顶峰之力的人。",
+	"skill.wom.ender_blast" : "末影爆裂",
+	"skill.wom.ender_blast.tooltip" : "“末地科技之力的巅峰。”\n\n这把枪由12个腔室组成。\n动能将被储存在其中并转化为末影之力。\n当扳机被扣下时，转化的末影之力的一半凝聚成末影子弹。\n另一半被转移到粒子加速器中对末影之力进行过载充能直到将末影子弹以巨大的力量和极高的精度发射出去。\n所有这些都在短短的一瞬间内完成……令人叹为观止！",
+	"skill.wom.ender_fusion" : "末影融合",
+	"skill.wom.ender_fusion.tooltip" : "“末地科技之力的巅峰。”\n\n两把枪各由12个腔室组成，总计为24腔室。\n储存的动能在两者之间共享，并转化为末影之力。\n当扳机被扣下时，转化的末影之力的一半凝聚成末影子弹。\n另一半被转移到粒子加速器中对末影之力进行过载充能直到将末影子弹以巨大的力量和极高的精度发射出去。\n所有这些都在短短的一瞬间内完成……令人叹为观止！",
+	
+	"item.wom.herrscher" : "君主之剑",
+	"item.wom.herrscher.tooltip" : "锋利而轻盈的剑在战斗中表现的非常出色....\n但......它似乎需要某些东西才能释放其全部潜力。",
+	"item.wom.gesetz" : "律法之盾",
+	"item.wom.gesetz.tooltip" : "强大而坚固的盾牌可以挡住最强烈的攻击......\n但......它似乎需要某些东西才能释放其全部潜力。",
+	"skill.wom.regierung" : "二重统治",
+	"skill.wom.regierung.tooltip" : "“攻守兼备，完美和谐。”\n格挡反击将根据你的格挡方式造成伤害。\n每次盾牌在玩家面前时，如果你受到攻击，你将反击。",
+	
+	"item.wom.moonless" : "无月",
+	"item.wom.moonless.tooltip" : "无柄之刃，无影之光······无月之夜。\n无月有明月与暗月两套攻击模式\n冲刺攻击可以在它们之间切换。",
+	"skill.wom.lunar_eclipse" : "月食",
+	"skill.wom.lunar_eclipse.tooltip" : "斩断光芒，收割生命······散播绝望。\n空袭你面前的敌人并让他们发光7秒\n明月模式下:\n击中发光的目标会叠加你造成的伤害并增加被击中目标的发光等级\n击中失明的目标会叠加你造成的伤害并移除被击中目标的失明效果\n暗月模式中:\n击中发光的目标会叠加你造成的伤害并移除被击中目标的发光效果\n击中失明的目标会叠加你造成的伤害并增加被击中目标的失明等级\n明月模式跳跃攻击(新月残光):释放对目标所有叠加的伤害\n如果它杀死了目标，将会反射到最近的目标，并减少5%的伤害\n发光会增加反射范围，失明会增加伤害\n暗月模式跳跃攻击(满月凌空):释放对目标叠加的70%伤害(AOE)\n失明会增加伤害",

```

```suggestion
	"skill.wom.lunar_eclipse.tooltip" : "“斩断光芒，收割生命······散播绝望。”\n空袭你面前的敌人并让他们发光7秒\n明月模式下：\n击中发光的目标会叠加你造成的伤害并增加被击中目标的发光等级\n击中失明的目标会叠加你造成的伤害并移除被击中目标的失明效果\n暗月模式下：\n击中发光的目标会叠加你造成的伤害并移除被击中目标的发光效果\n击中失明的目标会叠加你造成的伤害并增加被击中目标的失明等级\n明月模式跳跃攻击（新月残光）：释放对目标所有叠加的伤害\n如果它杀死了目标，将会反射到最近的目标，并减少5%的伤害\n发光会增加反射范围，失明会增加伤害\n暗月模式跳跃攻击（满月凌空）：以AOE释放对目标叠加的70%伤害。\n失明会增加伤害。",
```
`空袭你面前`的空袭是怎么翻的？

标点符号问题 全文通查（就改了一部分
加戏太多 全文自查